scripts/debian/start_docker_debian_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -x
+set -v
+echo "Starting start_docker.sh"
+TEST_NAME="DebianTest"
+echo "start docker test: " $TEST_NAME
+
+TEST_DIR=$(mktemp -d)
+trap "rm -rf $TEST_DIR" 0
+echo "test dir: " $TEST_DIR
+
+#set TMPDIR for aptly to use
+TMPDIR=$TEST_DIR
+KEY_DIR=$1
+echo "KEY_DIR: $KEY_DIR"
+echo "DC_IP: $DC_IP"
+STATUS=0
+
+# run the docker container
+docker \
+  run --rm --env-file ${HOME}/build_env_docker \
+  --mount type=bind,src=${GOPATH}/src/github.com/algorand/go-algorand/scripts/debian/,dst=/workdir \
+  --mount type=bind,src=${GOPATH}/src/github.com/algorand/go-algorand/test/e2e-go/cli/goal/expect,dst=/expectdir \
+  --mount type=bind,src=${GOPATH}/src/github.com/algorand/go-algorand/test/testdata,dst=/testdata \
+  --mount type=bind,src=${KEY_DIR},dst=/stuff \
+  -v type=bind,src=${TEST_DIR},dst=/testdir \
```

Comment:
do we even need /testdir ? if we just let files live within the docker context they'll be cleaned away when the container exits?

Reply:
Removed /testdir mount point, it was not necessary.